bundles/org.openhab.binding.smartthings/contrib/smartthings/SmartApps/OpenHabAppV2.groovy

```diff
@@ -889,6 +899,14 @@ def actionColor(device, attribute, value) {
             device.setColor(colormap)
             device.setLevel(value[2] as int)
         break
+        case "off":
+            // log.debug "actionColor: Setting device \"${device}\" with attribute \"${attribute}\" to off"
```

Comment:
Remove dead code

Reply:
I'm not sure what you want here. Is it to remove the commented out log.debug statement? If so I find it handy to leave those as written in case debugging is needed in the future. The Smartthings IDE does not have anything like an integrated debugger with nice things like breakpoints and variable examination.

Reply:
yes it's to remove commented out code. Can't you just leave it in, but not commented out?